doc/api/configuration.md

```diff
@@ -198,31 +198,17 @@ To unset:
 
 ## `feedback`
 
-Default: `True`
-
-Print number of rows affected by DML.
-
-```{code-cell} ipython3
-%config SqlMagic.feedback = True
+```{versionchanged} 0.10
+`feedback` takes values `0`, `2`, and `2` instead of `True`/`False`
```

Comment:
It's not very clear what `feedback` is. We can rephrase `quantity of feedback` to something more understandable

Reply:
`takes values 0, 1 , and 2` (2 is repeated twice)

Reply:
fixed https://github.com/ploomber/jupysql/pull/798/commits/1ab93c2432686d37782d39eebe364876090ba249